js/language_server/Makefile

```diff
@@ -23,11 +33,25 @@ package-lock.json:
 node_modules/.package-lock.json: package-lock.json
 	npm ci
 
+../tree-sitter/dist/tree-sitter.o:
+	$(MAKE) -C ../tree-sitter dist/tree-sitter.o
+
+dist/language-server-c-bindings.js: ../libyaml/dist/libyaml.o ../libpcre/dist/libpcre.a ../tree-sitter/dist/tree-sitter.o $(LANGUAGES:%=../languages/%/dist/semgrep-parser.o)
```

Comment:
nit: the naming (`c-bindings`) feels confusing to me, thoughts on `semgrep-language-server-wasm` or `semgrep-ls-wasm`?

Reply:
yea sure